build.sh

```diff
@@ -22,44 +22,72 @@ set -euo pipefail
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-ALL_APPS=()
-
-# Install linux dependencies for all apps
-for app in ${DIR}/vxsuite/apps/*; do
-  if [ -d "${app}" ]; then
-    tmp_dir=$(basename "${app}")
-    if [ -d "${app}/frontend" ]; then
-      make -C "${DIR}/vxsuite/apps/${tmp_dir}/frontend" install
+# Define vxsuite apps that can be built, along with the expected path prefix
+ALL_APPS=(admin central-scan mark scan)
```

Comment:
@kshen0 just added `mark-scan`, right?

```suggestion
ALL_APPS=(admin central-scan mark scan mark-scan)
```

Reply:
`mark-scan` was excluded in an earlier workaround, so I left it excluded. If @kshen0 can confirm it's ready to be included in this build process, happy to update.